modules/manage/partials/authentication.adoc

```diff
@@ -657,10 +657,18 @@ endif::[]
 
 Schema Registry and HTTP Proxy connect to Redpanda over the Kafka API. For the Kafka username and password, Redpanda uses ephemeral credentials internal to the cluster. Ephemeral credentials are regular SCRAM credentials, but they're only stored in memory and are lost when a broker restarts. When the Schema Registry or HTTP Proxy start up, they broadcast an ephemeral credential to other brokers over the internal RPC. If authentication fails to a particular broker, new ephemeral credentials are sent to that broker, and the service reconnects.
 
+[IMPORTANT]
```

Comment:
I am checking this part with the Kubernetes team to see what we currently do.

Either way, we'll need to update the paragraph before this section because we will no longer be using ephemeral credentials.

cc @chrisseto

Reply:
We should really be enabling auth on pandaproxy and kafka API together. The fact that most people don't use pandaproxy and just leave it alone probably means there are a bunch of clusters that have root access to their cluster exposed via pandaproxy.

> break some things...

Yeah, but it's for better security and TBH I really suspect very few people actually use the HTTP proxy for anything.